TradeItIosTicketSDK2/TradeItYahooTradePreviewViewController.swift

```diff
@@ -154,21 +157,42 @@ class TradeItYahooTradePreviewViewController: CloseableViewController, UITableVi
             return UITableViewCell()
         }
     }
-
+    
+    func tableView(_ tableView: UITableView, estimatedHeightForRowAt indexPath: IndexPath) -> CGFloat {
+        return UITableViewAutomaticDimension
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
```

Comment:
Should this be set every time or just once if it's the same for every cell?

https://developer.apple.com/library/content/documentation/UserExperience/Conceptual/AutolayoutPG/WorkingwithSelf-SizingTableViewCells.html

```swift
tableView.estimatedRowHeight = UITableViewAutomaticDimension
tableView.rowHeight = UITableViewAutomaticDimension
```